src/pages/apps/index.tsx

```diff
@@ -13,18 +13,21 @@ import useSafeAppsFilters from '@/hooks/safe-apps/useSafeAppsFilters'
 import SafeAppsFilters from '@/components/safe-apps/SafeAppsFilters'
 import { useHasFeature } from '@/hooks/useChains'
 import { FEATURES } from '@/utils/chains'
+import { useNativeSwapsAppCard } from '@/components/safe-apps/hooks/useNativeSwapsAppCard'
 
 const SafeApps: NextPage = () => {
   const router = useRouter()
+  const { swapsCardDetails } = useNativeSwapsAppCard()
   const { remoteSafeApps, remoteSafeAppsLoading, pinnedSafeApps, pinnedSafeAppIds, togglePin } = useSafeApps()
+  const allApps = useMemo(() => [swapsCardDetails, ...remoteSafeApps], [remoteSafeApps, swapsCardDetails])
```

Comment:
This can be reverted as well then?

Reply:
This is still needed to apply the filters

Reply:
ah just saw you other comment. I will remove the filtering then.

Reply:
I now have the swaps card being passed down separately from the safe apps list, and it disappears when a filter is applied